src/main/java/kg/jarkyn/server/Resource/PublicResource.java

```diff
@@ -0,0 +1,21 @@
+package kg.jarkyn.server.Resource;
+
+import java.io.File;
+import java.nio.file.Files;
+
+public class PublicResource {
```

Comment:
What's the difference and responsibility of a `PublicResource` and a `ResourceReader`. Just from the names I would have expected that `PublicResource` would _use_ a `ResourceReader` in on way or another.